examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>Each calibration is represented by a&nbsp;<code>PulseSequence</code>&nbsp;object that contains the OpenPulse implementation for the associated native gate. -&gt; Let's remove this statement since we have already mentioned that the calibration is represented by PulseSequence object. </p><p> </p><p>Retrieving a gate in particular can be done by passing the&nbsp;<code>Gate</code>&nbsp;object (instantiating with a numerical value or a&nbsp;<code>FreeParameter</code>&nbsp;if necessary) and the qubit of interest. For instance, you can inspect the pulse implementation of the XY(θ) with the code in next cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>